test/integration/suite_test.go

```diff
@@ -105,6 +105,7 @@ var _ = BeforeSuite(func() {
 	By("Setup custom controllers")
 	opts := options.NewOptions()
 	opts.LeaderElection.LeaderElect = false
+	opts.EnableV1Alpha1APIs = true
```

Comment:
the default is true so this is technically not needed?

Reply:
default was false before, now it's changed to true.